### ISSUE TITLE
Prereg form should allow for A/B testing

### ISSUE THREAD

**Issue by EliAndrewC** [magfest/ubersystem#9]
We occasionally debate whether to include any of the optional form fields on our prereg form.  There's been a lot of research to suggest that adding more form fields decreases conversion percentages, but we like having that extra data.

So it would be nice to have multiple versions of the prereg form both with and without the extra fields.  I've heard really good things about https://www.optimizely.com/ for making this happen.

---
- Bitbucket: https://bitbucket.org/EliAndrewC/magfest/issue/9
- Originally Reported By: [ ](http://bitbucket.org/EliAndrewC)
- Originally Created At: 2013-04-08T02:20:53.913

**Comment by bds002** [comment]
Should this be a closed issue based on Eli's comment?

**Comment by kitsuta** [comment]
Ehh, I'd say that integration with something like optimizely would still be Nice To Have. For example, we could try to use it to figure out:
* What makes people sign up to volunteer
* The impact of the badge buttons on people's willingness to buy certain kick-in levels
* What workflows encourage people to finish registering (as opposed to getting halfway through, closing the page, and then wondering why they don't have a badge)

**Comment by EliAndrewC** [comment]
Yeah, the more I think about it, the more I think this would still be a nice thing to have.  I could see closing it out anyway on the basis that I can't imagine us getting to it anytime in the forseeable future, but there's definitely some value we could capture.
